testing-modules/assertion-libraries/src/main/java/com/baeldung/assertj/extracting/Address.java

```diff
@@ -0,0 +1,25 @@
+package com.baeldung.assertj.extracting;
+
+class Address {
+    private String street;
```

Comment:
Couldn't be final?

Reply:
yes, definitely, but keeping it concise and short for the blog post snippets